tests/model_registry/model_catalog/utils.py

```
@@ -814,3 +816,207 @@ def get_metadata_from_catalog_pod(model_catalog_pod: Pod, model_name: str) -> di
+        files: Dictionary mapping form field names to (filename, content, mime_type) tuples
+
+    Returns:
+        dict: Parsed JSON response
+
+    Raises:
+        HTTPError: If response status is not successful
+    """
+    headers = {"Authorization": f"Bearer {token}"}
+
+    LOGGER.info(f"Executing model catalog POST: {url}")
+    response = requests.post(url=url, headers=headers, files=files, verify=False, timeout=60)
```

Ruff (0.14.10)

Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)